src/applications/mhv-secure-messaging/tests/e2e/keyboard-nav-test/secure-messaging-keyboard-nav-to-attachments-compose-page.cypress.spec.js

```diff
@@ -36,16 +36,13 @@ describe('Secure Messaging Keyboard Nav to Attachment', () => {
     // After closing the attachment banner, first attachment remove button has focus
     PatientComposePage.verifyRemoveAttachmentButtonHasFocus(0);
 
-    cy.injectAxe();
-    cy.axeCheck(AXE_CONTEXT, {
-      rules: {
-        'aria-required-children': {
-          enabled: false,
-        },
-      },
-    });
     PatientComposePage.sendMessage();
     PatientComposePage.verifySendMessageConfirmationMessageText();
     PatientComposePage.verifySendMessageConfirmationMessageHasFocus();
+
+    PatientComposePage.verifyHeader('Inbox');
```

Comment:
@fazilqa I am not sure this test covers the loading spinner validation, is that incorrect?

Reply:
this was added to prevent test failing.

Reply:
Are we planning to include Cypress coverage to test the spinner?

Reply:
no we don't